### PR TITLE
WIP - Expose deskew function

### LINKED ISSUE #28: Expose Deskew from Skew
Currently only findSkew is exposed in Skew.java , Deskew is also equally important which is not exposed by this lib. Kindly make available them. 

`PIX *  pixDeskew (PIX *pixs, l_int32 redsearch)`
`PIX *  pixFindSkewAndDeskew (PIX *pixs, l_int32 redsearch, l_float32 *pangle, l_float32 *pconf)`

### PR DESCRIPTION
**Pull request by SubhamTyagi**
- [x] `PIX * pixDeskew (PIX *pixs, l_int32 redsearch)`
- [ ] `PIX * pixFindSkewAndDeskew (PIX *pixs, l_int32 redsearch, l_float32 *pangle, l_float32 *pconf)`
- [ ]  Documentation 

Fixes #28

### CODE REVIEW

**Comment by Robyer** [comment]
@SubhamTyagi Thanks for working on this.

Please make sure to:
1) Respect original formatting of changed files. Don't reformat other lines and keep your new lines consistent with rest of the modified file.
2) Don't commit unrelated changes in this PR. That means changes to .idea files or build.gradle shouldn't be here. Commit only relevant changes.

So edit your commit to keep it clean (don't add new "cleanup" commit on top of this one, but ammend/squash them together).

File: tesseract4android/src/main/java/com/googlecode/leptonica/android/Skew.java [hunk 1]
```diff
@@ -102,11 +103,20 @@ public static float findSkew(Pix pixs, float sweepRange, float sweepDelta, int s
...
+	}
+
 	// ***************
 	// * NATIVE CODE *
 	// ***************
 
 	private static native float nativeFindSkew(long nativePix, float sweepRange, float sweepDelta,
 											   int sweepReduction, int searchReduction, float searchMinDelta);
 
+
```
Comment (Robyer) [inline tesseract4android/src/main/java/com/googlecode/leptonica/android/Skew.java:120]: There should be only single empty line between the methods to make it consistent.

**Comment by SubhamTyagi** [comment]
> Please make sure to:
> 
>     1. Respect original formatting of changed files. Don't reformat other lines and keep your new lines consistent with rest of the modified file.
> 
>     2. Don't commit unrelated changes in this PR. That means changes to .idea files or build.gradle shouldn't be here. Commit only relevant changes.
> 
> 
> So edit your commit to keep it clean (don't add new "cleanup" commit on top of this one, but ammend/squash them together).

1. okay 
2. .idea file should be ignored in git

**Comment by Robyer** [comment]
@SubhamTyagi Thanks for your idea and effort. In the end I implemented the function a bit differently (also with the support for getting the sweep/confidence result).

**Pull request closed without merging**